benches/aead.rs

```diff
@@ -0,0 +1,188 @@
+#![allow(missing_docs)]
```

Comment:
Please copy over the license block from src/lib.rs to this file. That would what the license block that crypto-bench's aead.rs should have had.

Reply:
It has been added.